dvc/command/experiments.py

```diff
@@ -1249,6 +1250,12 @@ def add_parser(subparsers, parent_parser):
         action="store_true",
         help="Remove all committed experiments.",
     )
+    remove_group.add_argument(
+        "-r",
+        "--git-remote",
```

Comment:
I don't think we should use `-r` here since that's used for `--remote` in other commands (possibly here later?)
I think argument/flag consistency is a guideline in https://clig.dev/
How about `-g`? or nothing

Reply:
Can we use `--remote` here? because `dvc exp remove` act only on refs not caches.

Reply:
I would defer to @jorgeorpinel, but I think `--git-remote` would be better for consistency and clarity.